github/oidc.go

```diff
@@ -50,6 +50,15 @@ type OIDCToken struct {
 
 	// JobWorkflowRef is a reference to the current job workflow.
 	JobWorkflowRef string `json:"job_workflow_ref"`
+
+	// RepositoryID is the unique repository ID.
```

Comment:
Note: these values may be present in the payload, but the payload's format depends on the trigger and the payload is not always present. So I think it's useful to extract it in a simple way and expose it.